proposals/PluginsAudit2015/audit.md

```diff
@@ -0,0 +1,223 @@
+# Cordova Plugins
+
+**NOTE**: Data for this proposal was collected on October 28th, 2015. 
+
+## Battery-Status
+* 1 Month Downloads: 5116
+* Open Issues: 18
+* Open Pull Requests: 3
+* Spec: http://www.w3.org/TR/battery-status/ 
+* android 5 has the ability to grab battery info in uiwebview built in
+* isPlugged is useful but not being used much
+* Recommendation: Deprecate! 
+
+## Camera
+* 1 Month Downloads: 17503
+* Open Issues: 85
+* Open Pull Requests: 20
+* Recommendation: Keep. Continue merging PRs and avoiding adding too many new features
+
+## Console
+* 1 Month Downloads: 45894
+* Open Issues: 5
+* Open Pull Requests: 1
+* console output should show up in Xcode by default. 
+* Recommendation: Move functionality back into platforms
+
+## Contacts
+* 1 Month Downloads: 6096
+* Open Issues: 71
+* Open Pull Requests: 10
+* Spec: http://www.w3.org/TR/contacts-api/ (discontinued)
+* API needs to be redone. Currently brittle and many platform specific quirks
```

Comment:
Can you provide an example of brittleness? I'm curious what the perceived failings are.